src/js/components/SiteHeader.tsx

```diff
@@ -62,7 +62,9 @@ const SiteHeader = () => {
 
   const changeLanguage = () => {
     const newLang = LNG_CHANGE[i18n.language];
-    const path = (location.pathname + location.search).replace(`/${i18n.language}/`, `/${newLang}/`);
+    // Can't rely on there being a trailing slash at the base page (.e.g, `/en` and `/en`/ are both valid), and can't
+    // ensure project IDs don't begin with an `en` or `fr`. Thus, we use a RegExp for language changing in the URL.
```

Comment:
How would it be ensure that projectIDs don't being with a lang

Reply:
regex with a `^` to mean beginning of the string.